locale/en.po

```diff
@@ -1211,15 +1211,15 @@ msgstr "Exit"
 
 #: ../src/general/StringUtils.cs:18
 msgid "BILLION_ABBREVIATION"
-msgstr "B"
+msgstr "{0} B"
```

Comment:
This is going to hurt a bit, all the translations that were already done, need to be cleared out, otherwise they'll cause issues. It can be done on translate.revolutionarygamesstudio.com but it will be a bit of a pain (I can do it if no one else wants to).